util/tooltool/tooltool.py

```diff
@@ -833,39 +836,39 @@ def upload(manifest, message, base_urls, auth_file, region):
     # Upload the files, each in a thread.  This allows us to start all of the
     # uploads before any of the URLs expire.
     threads = {}
-    for filename, file in files.iteritems():
-        if 'put_url' in file:
-            log.info("%s: starting upload", filename)
+    for filename, file_ in files.iteritems():
+        if 'put_url' in file_:
+            log.info("%s: starting upload", (filename,))
```

Comment:
I restored this to the original functionality [here](https://github.com/MozillaSecurity/funfuzz/commit/5922971e63f471851a54ad8b40550b12eed1b5f6#diff-9018c66b02761bd8e129f084cfae1539L853).